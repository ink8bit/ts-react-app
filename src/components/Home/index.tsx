import type { ReactElement } from 'react';

import React from 'react';

import s from './styles.css';

function Main(): ReactElement {
  return (
    <>
      <h1 className={s.title}>Home</h1>
      <div>content</div>
    </>
  );
}

export default Main;
