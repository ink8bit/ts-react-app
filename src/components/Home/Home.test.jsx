import React from 'react';
import { create } from 'react-test-renderer';

import Home from './index';

test('snapshot', () => {
  const c = create(<Home />);
  expect(c.toJSON()).toMatchSnapshot();
});
