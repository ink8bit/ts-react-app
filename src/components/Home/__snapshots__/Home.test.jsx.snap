// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
[
  <h1
    className="title"
  >
    Home
  </h1>,
  <div>
    content
  </div>,
]
`;
